src/Microsoft.Health.Dicom.Api/Controllers/StoreController.cs

```diff
@@ -40,6 +41,8 @@ public StoreController(IMediator mediator, ILogger<StoreController> logger)
         }
 
         [AcceptContentFilter(new[] { KnownContentTypes.ApplicationDicomJson }, allowSingle: true, allowMultiple: false)]
+        [Produces(KnownContentTypes.ApplicationDicomJson)]
+        [Consumes(KnownContentTypes.ApplicationDicom, new[] { KnownContentTypes.MultipartRelated + TypeSeperator + KnownContentTypes.ApplicationDicom })]
```

Comment:
Also, do we consume just JSON or do we also consume images too?

Reply:
https://github.com/microsoft/dicom-server/blob/main/docs/resources/conformance-statement.md#store-stow-rs
I think we only support consumption of application/dicom and multipart/realted